populate_db.py

```diff
@@ -316,13 +323,32 @@ def create_admin_message_settings():
 
 def create_custom_placeholders():
     custom_placeholder, _ = get_or_create(
-        CustomPlaceholder, name='Hills',
+        CustomPlaceholder, name='abc',
```

Comment:
Why? 
and yes, it looks like this PR is failing the build.

Reply:
My bad, I left it in place after doing the test.
I'll fix the tests.